manifest.php

```diff
@@ -44,7 +44,7 @@
     'label' => 'Proctoring',
     'description' => 'Proctoring for deliveries',
     'license' => 'GPL-2.0',
-    'version' => '7.8.3',
+    'version' => '7.8.4',
```

Comment:
The `StorageManager` has been introduced on a recent version of `taoQtiTest` (`~17.12.0`). So please increase both feature number and the dependency to taoQtiTest

Reply:
I would say the version has already been upgraded, but it seems there was a merge issue and the dependency was not updated when the conflict on the main PR has been resolved.
So I will fix that.